docs/linting/TROUBLESHOOTING.md

```diff
@@ -205,6 +205,25 @@ For example:
 
 See [this issue comment](https://github.com/typescript-eslint/typescript-eslint/issues/4102#issuecomment-963265514) for more details.
 
+## Changes to one file are not reflected when linting other files in my IDE
+
+> tl;dr: Restart your ESLint server to force an update.
+
+ESLint currently does not have any way of telling parsers such as ours when an arbitrary file is changed on disk.
+That means if you change file A that is imported by file B, it won't update lint caches for file B -- even if file B's text contents have changed.
+Sometimes the only solution is to restart your ESLint editor extension altogether.
+
+See [this issue comment](https://github.com/typescript-eslint/typescript-eslint/issues/5845#issuecomment-1283248238 'GitHub issue 5845, comment 1283248238: details on ESLint cross-file caching') for more information.
+
+:::tip
+[VS Code's ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) provides an `ESLint: Restart ESLint Server` action.
+:::
```

Comment:
@tkdodo I can't request your review formally, but still would love your input since you'd asked on Twitter 😄

Reply:
Thank you. This clarifies that it wasn't my Editor's fault (I'm using IntelliJ).